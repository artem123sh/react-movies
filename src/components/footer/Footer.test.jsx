import React from 'react';
import { render } from '@testing-library/react';
import Footer from './Footer';

describe('Footer', () => {
  test('renders Footer\'s snapshot', () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
});
