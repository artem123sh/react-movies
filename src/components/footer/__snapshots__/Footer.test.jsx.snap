// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders Footer's snapshot 1`] = `
<DocumentFragment>
  <footer
    class="sc-AxirZ jFmNHz"
  >
    <span
      class="sc-AxjAm hgfYjl"
    >
      netflix
    </span>
    <span
      class="sc-AxjAm cIZlLY"
    >
      roulette
    </span>
  </footer>
</DocumentFragment>
`;
